cmd/validate/image_test.go

```diff
@@ -211,8 +214,8 @@ func Test_ValidateImageCommand(t *testing.T) {
 			"containerImage": "registry/image:tag",
 			"violations": [],
 			"warnings": [],
-			"success": true,
-			"successCount": 14
+			"successes": [],
+			"success": true
```

Comment:
Is this right? I thought we said that if there are no successes, no warnings, and no failures, then it should fail.

Reply:
Updated the test. The result mocked from Conftest had `Successes: 14` but no
shadowed `Successes` array of results, that meant that the output lacked
`successes` but did not fail as the total number of rules was `14`. I've added
one successful result and set `14` to `1`.